change/@fluentui-react-87fd4c4f-594a-4a62-b5a2-8306eb0c1d19.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This got generated in the wrong format somehow. Can you delete this file, merge with master, and then run `yarn` and `yarn change` again?

Reply:
@ecraig12345 Regenerated now. :)